BoxContentSDK/BoxContentSDK/Models/BOXMetadata.h

```diff
@@ -28,6 +28,18 @@
  */
 @property (nonatomic, readwrite, strong) NSString *scope;
 
+/**
+ * The current version of the metadata.
+ */
+@property (nonatomic, readwrite, strong) NSNumber *version;
+
+
+/**
+ * The current type version of the metadata.
```

Comment:
Can we better explain what a "type version" is? Is it the version of the type or something else?